fastapi_security/basic.py

```diff
@@ -36,3 +53,47 @@ def _make_credentials(
             c if isinstance(c, HTTPBasicCredentials) else HTTPBasicCredentials(**c)
             for c in credentials
         ]
+
+
+class BasicAuthWithDigestValidator:
```

Comment:
```suggestion
class BasicAuthWithDigestValidator(BasicAuthValidator):
```

Reply:
`mypy` does not like the idea of having two different `init` methods:

```
fastapi_security/basic.py:46: error: Signature of "init" incompatible with supertype "BasicAuthValidator"
```

A way to solve this would be to introduce a common base class from which plaintext- and digest-based basic auth validators would inherit. Let me know if you'd like that.